purchase_order_receipt_note/reports/purchase_order_receipt_templates.xml

```diff
@@ -0,0 +1,180 @@
+                />
+            </t>
+            <t t-if="o.dest_address_id">
+                <t t-set="information_block">
+                    <strong>Shipping address:</strong>
+                    <div t-if="o.dest_address_id">
+                        <div
+                            t-field="o.dest_address_id"
+                            t-options='{"widget": "contact", "fields": ["address", "name", "phone"], "no_marker": True, "phone_icons": True}'
+                            name="purchase_shipping_address"
+                        />
+                    </div>
+                </t>
+            </t>
+            <t t-else="">
+                <t t-set="information_block">
+                    <strong>Shipping address:</strong>
+                    <div t-if="o.picking_type_id and o.picking_type_id.warehouse_id">
+                        <span t-field="o.picking_type_id.warehouse_id.name" />
+                        <div
+                            t-field="o.picking_type_id.warehouse_id.partner_id"
+                            t-options='{"widget": "contact", "fields": ["address", "phone"], "no_marker": True, "phone_icons": True}'
+                        />
+                    </div>
+                </t>
+            </t>
+            <div class="page">
+                <div class="oe_structure" />
+                <div class="mt-4">
+                    <h2>Receipt Note #<span t-field="o.name" /></h2>
+                </div>
+                <div id="informations" class="row mt-4 mb32">
+                    <div t-if="o.user_id" class="col-3 bm-2">
+                        <strong>Purchase Representative:</strong>
+                        <p t-field="o.user_id" class="m-0" />
+                    </div>
+                    <div t-if="o.partner_ref" class="col-3 bm-2">
+                        <strong>Vendor Ref:</strong>
+                        <p t-field="o.partner_ref" class="m-0" />
+                    </div>
+                    <div
+                        t-if="o.state in ['purchase','done'] and o.date_approve"
+                        class="col-3 bm-2"
+                    >
+                        <strong>Order Date:</strong>
+                        <p t-field="o.date_approve" class="m-0" />
+                    </div>
+                    <div t-elif="o.date_order" class="col-3 bm-2">
+                        <strong>Order Deadline:</strong>
+                        <p t-field="o.date_order" class="m-0" />
+                    </div>
+                </div>
+                <table class="table table-sm o_main_table table-borderless mt-4">
+                    <thead style="display: table-row-group">
+                        <tr>
+                            <th name="th_description"><strong>Description</strong></th>
+                            <th name="th_internal_reference"><strong
+                                >Buyer Part No.</strong></th>
```

Comment:
I suppose it should be styled like this. Please do the same for the other `th` tags.
```suggestion
                            <th name="th_internal_reference">
                                <strong>Buyer Part No.</strong>
                            </th>
```